terasoluna-gfw-common-libraries/terasoluna-gfw-common/src/main/java/org/terasoluna/gfw/common/message/ResultMessages.java

```diff
@@ -237,6 +241,42 @@ public static ResultMessages danger() {
         return new ResultMessages(DANGER);
     }
 
+    /**
+     * factory method for primary messages.
+     * @return error messages
```

Comment:
Is the return value of `primary / secondary / light / dark` added this time be` error message`?

Reply:
Thx! The problem is inherited because `danger()` returned `error messages`.
Please see https://github.com/terasolunaorg/terasoluna-gfw/pull/1009/commits/d4fef8790d1e3ecf6144d068fae641d0a45fe6b6